Cargo.toml

```diff
@@ -11,6 +11,9 @@ authors = ["Richard Newman <rnewman@twinql.com>", "Nicholas Alexander <nalexande
 [dev-dependencies.datomish-cli]
   path = "cli"
 
+[dev-dependencies.datomish-explorer]
+  path = "explorer"
+
 [[bin]]
 name = "datomish-cli"
```

Comment:
Might need to add it here to be able to run with `cargo run`.  Assuming the command would be `cargo run --bin datomish-explorer`.

Alternatively, maybe we could fold the cli portion of the explorer into the normal cli.  So `cli` would depend on `explorer` and explorer would be just the web server code.  Since the current cli does nothing, starting the explorer could be the default behavior